### ISSUE TITLE
Method Not Allowed

### ISSUE THREAD

**Issue by jackyrx** [nagadomi/waifu2x#114]
Web Usage

**Comment by nagadomi** [comment]
That is a browser bug.
As far as I know IE on Windows 10 has a that bug.
Please use Edge or other browser (I recommend Google Chrome).

**Comment by deideloo** [comment]
I don't think this is a bug. It happens on Opera and Google Chrome too. It means you probably just typed in "waifu2x.udp.jp/api". You can't just visit that page because the browser won't be sure what data to fetch for it; all of the URLs for upscaled images are the same. I hope I helped.

**Comment by nagadomi** [comment]
GET request is not allowed in waifu2x.udp.jp/api.
We won't provide a image URL. Use `Download` button or `save image` function in the browser.
